### ISSUE TITLE
Issue while running 'npm install' on downloaded application.

### ISSUE THREAD

**Issue by kunal-motiani** [ryanmurakami/pizza-luvrs#9]
Hi Ryan,

I am facing this issue on running npm install on the pizza-luvrs app

![image](https://user-images.githubusercontent.com/29423686/27119607-1fb0129a-50fe-11e7-9d57-45dd9521cfed.png)

I am not familiar with nodejs, Can you tell me if I am doing anything wrong in the setup?

**Comment by ryanmurakami** [comment]
Hi @kunal-motiani,
Oooh, that's weird. Haven't seen that before. Can you let me know the versions of node & npm you have installed?

Also, try running `npm cache clean` and then `npm install`.

**Comment by kunal-motiani** [comment]
Hi Ryan,

The issue was resolved after I uninstalled nodejs and reinstalled it. Earlier I was using the v6.11.0 LTS version of node. 

Currently, I installed the following versions. 
npm -version = 5.0.3
node -v = v8.1.1

![image](https://user-images.githubusercontent.com/29423686/27135456-2332e4fa-5136-11e7-9c2b-a733884a0168.png)

Looks like earlier I might not have installed npm package altogether as it wasn't the default selection,

But now its running fine :- 

![image](https://user-images.githubusercontent.com/29423686/27135545-6045c61e-5136-11e7-9c15-2a6a8c3debf6.png)


Thanks 👍
